<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QMAppCacheSettingsWidget</class>
 <widget class="QWidget" name="QMAppCacheSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Durch das Aktivieren dieser Einstellung wird eine lokale Kopie der Software auf dem Rechner erstellt und diese bei jedem Start aktuell gehalten. Wenn die eigentliche Software auf einem Netzlaufwerk liegt, kann die Maßnahme den Start der Software beschleunigen. Der Start der Software wird dann aber trotzdem abgebrochen, wenn keine Verbindung zum Quellordner zur Überprüfung hergestellt werden kann, um sicherzustellen, dass nur mit der aktuellsten Softwareversion auf eine Datenbank zugegriffen wird und diese nicht beschädigt wird.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbAppCache">
     <property name="title">
      <string>&amp;Aktiviere lokale Anwendungskopie</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="2">
         <widget class="QPushButton" name="pbChooseSourceFolder">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>Auswählen</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Quellordner</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="leSourcePath">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Der Pfad zu dem Ordner in welchem die appcache.json und die ausführbare Datei liegen.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Zielordner</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="leTargetPath">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="pbChooseTargetFolder">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>Auswählen</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Der Pfad zu dem Ordner, in welchem zukünftig die appcache.json und die ausführbare Datei liegen.</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbChooseSourceFolder</sender>
   <signal>clicked()</signal>
   <receiver>QMAppCacheSettingsWidget</receiver>
   <slot>chooseSourceFolder()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>520</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbChooseTargetFolder</sender>
   <signal>clicked()</signal>
   <receiver>QMAppCacheSettingsWidget</receiver>
   <slot>chooseTargetFolder()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>520</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>chooseSourceFolder()</slot>
  <slot>chooseTargetFolder()</slot>
 </slots>
</ui>
